tests/core/test_state_sync.py

```diff
@@ -1248,13 +1264,17 @@ def test_unpause_snapshots_remove_intervals(
         ),
         version="a",
     )
+    snapshot.categorize_as(SnapshotChangeCategory.BREAKING)
+    snapshot.version = "a"
```

Comment:
In another context, I would get an error getting table info from these snapshots because they were not categorized. Therefore this categorizes them but then maintains the expected version. I believe this maintain the original intent of this test.